doc/conf.py

```diff
@@ -469,6 +469,15 @@ def add_js_css_files(app, pagename, templatename, context, doctree):
     "auto_examples/exercises/plot_cv_digits.py": (
         "auto_examples/model_selection/plot_nested_cross_validation_iris.py"
     ),
+    "auto_examples/cluster/plot_agglomerative_clustering.py": (
+        "auto_examples/cluster/plot_agglomerative_clustering_metrics.py"
+    ),
+    "auto_examples/cluster/plot_digits_agglomeration.py": (
```

Comment:
This example is clearly missing some narrative but I'm not sure that I would like to remove it. The fact that this is run on digits is quite different from just a feature selection on tabular data.

To me this is a nice example because it bring complementary intuition that you can easily visualize. However it should be improved because it does not describe what we should actually visualize.